examples/cpp_execution/text_inference_using_cpp.py

```diff
@@ -0,0 +1,251 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# -----------------------------------------------------------------------------
+import argparse
+import logging
+import os
+import sys
+from pathlib import Path
+from typing import List, Optional, Union
+
+from transformers import PreTrainedTokenizer, PreTrainedTokenizerFast
+
+import QEfficient
+from QEfficient.cloud.export import get_onnx_model_path
+from QEfficient.generation.text_generation_inference import fix_prompts, get_compilation_dims, get_input_prompts
+from QEfficient.utils import check_and_assign_cache_dir, get_qpc_dir_path, load_hf_tokenizer, qpc_exists
+from QEfficient.utils.logging_utils import logger
+
+script_dir = Path(__file__).resolve().parent
+so_folder_path = script_dir / "build"
+
+if so_folder_path.is_dir():
+    sys.path.append(str(so_folder_path))
+    try:
+        import InferenceSetIOBuffer  # noqa: E402
+    except ImportError:
+        logger.info("Error importing InferenceSetIOBuffer Module")
+else:
+    logger.info("so file's folder not found")
```

Comment:
raise error saying `FileNotFoundError("Please follow README instructions to first compile the cpp files")`